src/genome/gene/foodGene.cpp

```diff
@@ -0,0 +1,33 @@
+//
+// Created by Rosa on 1/12/2023.
+//
+
+#include "foodGene.h"
+
+FoodGene::FoodGene(GenomeView & view) :
+	foodType((FoodType) read4(view, 0)),
```

Comment:
Just curious what bonus is here. Is that what the organism gains from that food?